app/controllers/articles_controller.rb

```diff
@@ -138,6 +138,13 @@ def create
     @user = current_user
     article = Articles::Creator.call(@user, article_params_json)
 
+    check_embed_tag = @article.processed_html.split
+
+    if check_embed_tag[0] == "<p>Liquid"
```

Comment:
Definitely acknowledging the bug, but this solution won't work because it is quite believable to have the following:

```html
<p>Liquid is a non-evaluating templating framework developed by Shopify.</p>
```

Which would create an error.  We need to explore the solution deeper in the stack.  Namely when we attempt to parse a liquid declaration and it fails.

Reply:
Hi @jeremyf , I will take a look at the PR #16704  , I will investigate in depth where we can perfectly evaluate the `liquid tag`. appreciate your reply.